instrumentation/google.golang.org/grpc/otelgrpc/doc.go

```diff
@@ -13,23 +13,33 @@
 
-**notice**: **Do not use both interceptors and stats handlers at the same time!** If so, you will get duplicated spans and the parent/child relationships between spans will also be broken.
+We strongly still recommand you to use [stats.Handler], mainly for two reasons:
 
-Although the implementation `stats.Handler` in experimental stage, we strongly still recommand you to use `stats.Handler`, mainly for two reasons:
-- **Functional advantages**: `stats.Handler` has more information for user to build more flexible and granular metric, for example
-  - multiple different types of represent "data length": In [InPayLoad](https://pkg.go.dev/google.golang.org/grpc/stats#InPayload), there exists `Length`, `CompressedLength`, `WireLength` to denote the size of uncompressed, compressed payload data, with or without framing data. But in Interceptors, we can only got uncompressed data, and this feature is also removed due to performance problem. [#3168](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3168)
-  - more accurate timestamp: `InPayload.RecvTime` and `OutPayload.SentTime` records more accurate timestamp that server got and sent the message, the timestamp recorded by interceptors depends on the location of this interceptors in the total interceptor chain.
-  - some other use cases: for example [catch failure of decoding message](https://github.com/open-telemetry/opentelemetry-go-contrib/issues/197#issuecomment-668377700)
+Functional advantages: [stats.Handler] has more information for user to build more flexible and granular metric, for example
 
-- **Performance advantages**: If too many interceptors are registered in a service, the interceptor chain can become too long, which increases the latency and processing time of the entire RPC call.
+  - multiple different types of represent "data length": In [stats.InPayLoad], there exists "Length", "CompressedLength", "WireLength" to denote the size of uncompressed, compressed payload data, with or without framing data. But in interceptors, we can only got uncompressed data, and this feature is also removed due to performance problem.
+
+  - more accurate timestamp: [stats.InPayLoad]'s "RecvTime" and [stats.OutPayLoad]'s "SentTime" records more accurate timestamp that server got and sent the message, the timestamp recorded by interceptors depends on the location of this interceptors in the total interceptor chain.
+
+  - some other use cases: for example, catch failure of decoding message.
+
+Performance advantages: If too many interceptors are registered in a service, the interceptor chain can become too long, which increases the latency and processing time of the entire RPC call.
+
+[stats.Handler]: https://pkg.go.dev/google.golang.org/grpc/stats#Handler
+[stats.InPayLoad]: https://pkg.go.dev/google.golang.org/grpc/stats#OutPayLoad
+[stats.OutPayLoad]: https://pkg.go.dev/google.golang.org/grpc/stats#InPayload
```

Comment:
```suggestion
[stats.InPayload]: https://pkg.go.dev/google.golang.org/grpc/stats#OutPayload
[stats.OutPayload]: https://pkg.go.dev/google.golang.org/grpc/stats#InPayload
```

and also update the casing in the references